.travis.yml

```diff
@@ -64,6 +64,7 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[all]"
+  - "./examples/app-recreate.sh"
```

Comment:
Yes, I also think it's better not to introduce `app-recreate.sh` but rather to stick to "duplicating"  instructions in `test_example_app.py`, as we have it done elsewhere. 

One additional benefit is getting nice explicit documentation generated out of `app.py` docstring.

@hachreak Note that we had discussed using shell scripts in the past for testing example apps -- this was actually the very first version I had -- but after a brief discussion we have agreed to go for  the `test_example_app.py` all-Pythonic approach. I refactored the same thing in the past :smile: see  the debate in https://github.com/inveniosoftware/invenio-records/pull/90. Can you please also amend similarly?